object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -178,8 +184,14 @@ private void checkPseudoAssignToOwning(Node node) {
       } else {
         throw new BugInCF("unexpected AssignmentContext type " + assignmentContext.getClass());
       }
+    } else {
+      Node mustCallChoiceParam = getMustCallChoiceParam(node, defs);
```

Comment:
Can you add a comment to explain what is going on in this case?

Reply:
In this case, we are handling method invocation nodes that are not assigned to anything but the receiver of the method is in the defs. For example, we won't report a warning for "sockChannel.socket();" if sockChannel is a local variable in the defs.